src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp

```diff
@@ -903,34 +895,34 @@ Dictionary AdaptiveTileRendererFactory::get_params_metadata()
         "batch_size",
         Dictionary()
             .insert("type", "int")
-            .insert("default", "8")
+            .insert("default", "16")
             .insert("label", "Batch Size")
             .insert("help", "How many samples to render before each noise calculation"));
```

Comment:
What about "before each convergence estimation" instead of "before each noise calculation"?